14/README.md

```diff
@@ -134,28 +136,34 @@ For this BEP, we analyze a round-robin connection strategy, similar to [Round-ro
 In this scenario, every request is forwarded to a different node, distributing the load to all the nodes, and allowing recovering from connection errors by forwarding the failed request to another healthy node.
 
 This strategy needs to keep track of the following values:
-- `E`, the list of BigchainDB endpoints passed at initialization time (we use the Python notation `len(L)` to get the length of the list).
-- `i`, the index of the BigchainDB node to use for a request (`0 ≤ i < len(L)`).
```

Comment:
`i` is used in the definition of the algorithm below, but it is now removed. I'd restore this line.